src/spec-node/dockerCompose.ts

```diff
@@ -322,12 +333,16 @@ async function startContainer(params: DockerResolverParameters, buildParams: Doc
 			const files = configFiles?.split(',') ?? [];
 			const persistedBuildFile = await checkForPersistedFile(buildCLIHost, output, files, featuresBuildOverrideFilePrefix);
 			const persistedStartFile = await checkForPersistedFile(buildCLIHost, output, files, featuresStartOverrideFilePrefix);
-			if (persistedBuildFile && persistedStartFile) {
+			if ((persistedBuildFile.foundLabel && persistedBuildFile.fileExists) // require build file if in label
```

Comment:
Should this be `(!persistedBuildFile.foundLabel || persistedBuildFile.fileExists)`?

Reply:
Yes - thanks!